### DISCUSSION TITLE
spark load hive外表问题

### DISCUSSION THREAD

**Discussion by pushiwuhua7** [apache/doris#6676] [A - General / Q&A] [answered]
就是我用spark-load去导入
然后hive那边是有map类型的，但创建的外表不支持map类型
最后spark那边好像报map不能coat成varchar
请问怎么解决好
 java.lang.RuntimeException: Error while encoding: java.lang.RuntimeException: scala.collection.immutable.HashMap$HashTrieMap is not a valid external type for schema of string

**Comment by wangbo** [comment, marked as answer]
目前spark load目前不支持导入复杂类型的，得你自己在hive里转成基本类型

**Reply by pushiwuhua7** [reply]
好吧...
